examples/20_file_writer/main.neva

```diff
@@ -0,0 +1,17 @@
+import { io }
+
+const {
+  filename string = 'file_writer_example.txt'
```

Comment:
What will happen if file doesn't exist? will it throw error or create that file?